src/extension/utils.ts

```diff
@@ -99,9 +99,19 @@ export function getTerminalRunmeId(t: vscode.Terminal): string|undefined {
     ?? undefined
 }
 
+export function getCellRunmeId(cell: vscode.NotebookCell) {
+  return getCellUUID(cell)
+}
+
+function getCellUUID(cell: vscode.NotebookCell): string {
+  return getAnnotations(cell)['runme.dev/uuid']!
+}
```

Comment:
Why do we use two functions here that call each other?

Reply:
Because the runme id may change in the future.

For example, we may want to prefix it with the filename, like `${filename}:${uuid}`